inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -122,6 +123,7 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
     // WA: ConvertPriorBox must be executed before the 1st ConstantFolding pass
     manager.register_pass<ngraph::pass::ConvertPriorBox>();
     manager.register_pass<ngraph::pass::MVN6Decomposition>();
+    manager.register_pass<ngraph::pass::SimplifyCTCGreedyDecoder>();
```

Comment:
Why it is included only to CPU pipeline?

Reply:
Was added for cldnn and gna plugins. If need to add one more, let me know.

Reply:
So why it is not included into CommonOptimizations pipeline?